.circleci/config.yml

```diff
@@ -42,6 +42,7 @@ references:
 version: 2.1
 
 orbs:
+  hmpps: ministryofjustice/hmpps@8.0.3
```

Comment:
This can be changed just to `ministryofjustice/hmpps@8` so we don't need to update to minor versions 🙂